lib/validation/validators.js

```diff
@@ -85,21 +85,27 @@ module.exports = Validators = {
   },
 
   // eslint-disable-next-line no-inline-comments, spaced-comment
-  before(value/*, [diff, unit][, diff, unit]*/) {
-    const args = [].slice.call(arguments, 1);
-    let date = moment(value, dateFormat);
-    let diff;
-    let unit;
-    while (args.length) {
-      diff = args.shift();
-      unit = args.shift() || 'years';
-      date = date.add(diff, unit);
+  before(value, date/*, [diff, unit][, diff, unit]*/) {
```

Comment:
No worries, ultimately, this is where the documentation will be handy.

hmm I swear I tried that signature but it didn't work. I had to wrap it in an object with a type key e.g. 
`validate: [{type: 'before', arguments: [18, 'years', 11, 'months']}]`
I'll need to try it again and also update the document to reflect it.